$NetBSD: patch-bd,v 1.5 2014/08/21 07:49:56 wiz Exp $

Add MANDIR, but don't hardcode it (fixed in post-patch by SUBST_*).

--- mcs/build/config-default.make.orig	2014-08-09 11:10:40.000000000 +0000
+++ mcs/build/config-default.make
@@ -19,6 +19,7 @@ CFLAGS = -g -O2
 endif
 prefix = /usr/local
 exec_prefix = $(prefix)
+mandir = $(prefix)/MANDIR
 mono_libdir = $(exec_prefix)/lib
 sysconfdir = $(prefix)/etc
 #RUNTIME = mono
